src/box/iproto.cc

```diff
@@ -1883,17 +1890,19 @@ tx_accept_msg(struct cmsg *m)
 	msg->connection->iproto_thread->tx.requests_in_progress++;
 	rmean_collect(msg->connection->iproto_thread->tx.rmean,
 		      REQUESTS_IN_PROGRESS, 1);
+	msg->response_begin = obuf_create_svp(msg->connection->tx.p_obuf);
```

Comment:
I don't think you need `msg->response_begin`, because you pass `svp` that points to the beginning of a response to `tx_end_msg`, anyway.

Reply:
It was an artifact from first patch version (sorry, pushed patch at the end of day, so didn't check it properly). Removed.